lms/templates/fields/field_textarea.underscore

```diff
@@ -13,10 +17,12 @@
         var textareaDescribedBy = (message ? 'u-field-message-help-' : 'u-field-placeholder-value-') + id;
```

Comment:
Applies to line above this one, the aria-labelledby is unnecessary in read-only context.